### ISSUE TITLE
Menubar and dock in OS X Yosemite

### ISSUE THREAD

**Issue by johanberonius** [arp/old-lion-fullscreen#2]
Since OS X Yosemite the fullscreen mode doesn't fill the entire screen anymore. There are still space reserved for the menu bar at the top and the dock at the bottom. Even if dock is set to auto hide there are a few pixels left.
On a secondary screen if there is no menu bar this is not an issue but on the main screen it is not quite fullscreen.
It also affects the "HTML5 Fullscreen API"-mode.
![old-lion-fullscreen-osx-yosemite](https://cloud.githubusercontent.com/assets/722210/4779664/291eea98-5c3e-11e4-9b16-46c02380f784.png)

It is still a lot better than the default fullscreen mode that doesn't auto hide all toolbars and blacks out all other screens though.

**Comment by HenryHatten** [comment]
I'm seeing the exact same behaviour as johanberonius in Yosemite also. Is anybody working on this anymore I wonder? It'd be a pity if not as its a great extension.

**Comment by eriktar** [comment]
Just adding my whish to get this fixed as well.  I'm so used to this on my 11" Mac Book Air that I find firefox more or less useless without this plugin.  Please fix. :)

**Comment by jdahdah** [comment]
Same issue here. Not a developer, so I can't fix it myself. It would be nice to hear at least a word of acknowledgement from the original developer.

**Comment by johanberonius** [comment]
I don't think this is fixable within the scope of this add-on since all it does is toggles a feature within Firefox. (This is the effective line https://github.com/arp/old-lion-fullscreen/blob/master/chrome/content/old-lion-fullscreen.xul#L7)

For comparison, Chrome has both a "Fullscreen" and a "Presentation" mode that only differs in whether they hide the address and tool bars. They both black out any other screens making them useless.

**Comment by johanberonius** [comment]
This seems to be fixed with Firefox 41. Now it is truly full screen, nice!
